platforms/android/library/src/androidTest/java/io/element/android/wysiwyg/inputhandlers/InterceptInputConnectionIntegrationTest.kt

```diff
@@ -57,7 +58,7 @@ class InterceptInputConnectionIntegrationTest {
             textView.text.dumpSpans(), equalTo(
                 listOf(
                     "hello: android.widget.TextView.ChangeWatcher (0-5) fl=#6553618",
-                    "hello: android.text.style.StyleSpan (0-5) fl=#33",
+                    "hello: android.text.style.StyleSpan (0-5) fl=#17",
```

Comment:
These changes are caused by using `inSpans`, but they shouldn't affect the functionality of the editor.